executor/Dockerfile.build

```diff
@@ -10,4 +10,4 @@ RUN pip install -r requirements.txt
 
 COPY . /opt/cook
 
-CMD ["pyinstaller", "-F", "-n", "cook-executor", "-p", "cook", "cook/__main__.py"]
+CMD ["pyinstaller", "--runtime-tmpdir", ".cook-executor", "-F", "-n", "cook-executor", "-p", "cook", "cook/__main__.py"]
```

Comment:
What is this relative to? Should we make it absolute?

Reply:
(same question for the other places)

Reply:
We want it to be relative to the sandbox directory, we don't have access to that path while running this command. I can add a comment in the Cook executor README if that helps.